README.md

```diff
@@ -1,12 +1,22 @@
 
 # Casacore
 
-A suite of c++ libraries for radio astronomy data processing.
+A suite of C++ libraries for radio astronomy data processing.
 
 
 # Installation
 
-## Obtaining the source
+## Debian / Ubuntu
```

Comment:
We could add a section for macOS, something like:

## macOS

Use Homebrew with the `ska-sa` tap to install casacore:

```bash
$ brew tap ska-sa/tap
$ brew install casacore
```